lms/envs/bok_choy.py

```diff
@@ -253,7 +253,6 @@
 
 # Make sure we test with the extended history table
 FEATURES['ENABLE_CSMH_EXTENDED'] = True
-INSTALLED_APPS.append('coursewarehistoryextended')
```

Comment:
Is this installed elsewhere now?

Reply:
This is installed in the bok_choy environment by defaulting ENABLE_CSMH_EXTENDED to true.